chroma-agent/chroma-agent.spec

```diff
@@ -131,6 +131,8 @@ sed -ie 's/^SELINUX=.*$/SELINUX=disabled/' /etc/selinux/config
 # the above only disables on the next boot.  set to permissive currently, also
 setenforce 0
 
+rm -f /var/lib/iml/zfs_store.json
```

Comment:
Do we only want to do this on upgrade?

Reply:
what about if someone installs, uninstalls then installs the new version? that's not an upgrade but they will still have the file because I'm not sure it got removed on uninstall. can change it if you would like?

Reply:
sounds good to keep it